tests/index-test.ts

```diff
@@ -31,8 +30,8 @@ describe('SonarQube Formatter', () => {
         },
       },
     });
-    debugger;
-    let result = await runBin();
+
+    let result = await runBin('.', '--format', require.resolve('..'));
```

Comment:
Any reason we moved this from `staticArgs` above?

Reply:
> Any reason we moved this from `staticArgs` above?

In [line 177](https://github.com/ember-template-lint/ember-template-lint-formatter-sonarqube/pull/131/files#diff-b573cbcdde0af75348b2bc182d9e499228fc060589c4211318d0bc21cf03cdcbR177) we need to pass other args to runBin instead of these "static" args

Reply:
Ah ok, makes sense. It may be nice for us to `require.resolve` the formatter into a local variable and use that in each invocation.

```suggestion
    // somewhere up above...
    const formatterPath = require.resolve('..');

    let result = await runBin('.', '--format', formatterPath);
```

Reply:
Updated!